api_service.py

```
@@ -2055,6 +2052,25 @@ async def startup_event():
+        try:
+            logger.info("Starting background processor (delayed start after server ready)...")
+            start_background_processor()
+            logger.info("Background processor started successfully")
+        except Exception as e:
+            logger.error(f"Failed to start background processor: {e}")
+            logger.exception("Full traceback:")
+            # Don't exit - allow the API to run even if background processor fails
+            logger.warning("API service will continue without background processing")
+
+    # Schedule the delayed start as a background task
+    asyncio.create_task(delayed_background_start())
```

Ruff (0.14.3)

Store a reference to the return value of `asyncio.create_task`

(RUF006)